blueprint/overview.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: Solution overview
 ---
 
-This overview covers the blueprint's purpose, its components, and design considerations and decisions. It also maps blueprint solutions to Essential Eight security strategies.
+This overview covers the blueprint's purpose, its components, and design considerations and decisions. It also maps the blueprint solutions to the Essential Eight maturity model.
```

Comment:
I actually think the second sentence can be taken out as the E8 table now lives in its own page.

Reply:
Agreed, the sentence has been removed.